src/sentry/interfaces/contexts.py

```diff
@@ -135,7 +135,7 @@ class MonitorContextType(ContextType):
 @contexttype
 class TraceContextType(ContextType):
     type = "trace"
-    indexed_fields = {}
+    indexed_fields = {"": "{id}", "sample_rate": "{sample_rate}"}
```

Comment:
do we want to update DS_DENYLIST https://github.com/getsentry/sentry/blob/f9bf4e27bab9fcb03e32dbf6dbc8418608f7f247/src/sentry/constants.py#L600-L627?
cc @jan-auer

Reply:
I believe those fields are dynamic-sampling fields which are a distinct concept from fields one can query in snuba